CAMEL-19687 - Removed Camel-Atlasmap-Starter

oscerd opened the pull request.

davsclaus commented:
I didnt spot if you also deleted the itest

tests/camel-itest-spring-boot/src/test/java/org/apache/camel/itest/springboot/CamelAtlasmapTest.java

oscerd commented:
> I didnt spot if you also deleted the itest
> 
> tests/camel-itest-spring-boot/src/test/java/org/apache/camel/itest/springboot/CamelAtlasmapTest.java

Yes, I removed it too.

oscerd commented:
Errors in tests are not related.
